You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard deviation is 209. How well did you score on the test compared to the average test taker?

In [1]:
mu = 1026
x = 1100
sd = 209

z = (x-mu)/sd
z

0.35406698564593303

z = 0.35406698564593303
scored more .35 standard deviation of the mean 1026.
z value from table is .6368
there are 63% of test takers below you